x-pack/plugin/inference/qa/mixed-cluster/src/javaRestTest/java/org/elasticsearch/xpack/inference/qa/mixed/AzureOpenAiServiceMixedIT.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.inference.qa.mixed;
+
+import org.apache.lucene.tests.util.LuceneTestCase;
+import org.elasticsearch.Version;
+import org.elasticsearch.common.Strings;
+import org.elasticsearch.inference.TaskType;
+import org.elasticsearch.test.http.MockResponse;
+import org.elasticsearch.test.http.MockWebServer;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+import static org.elasticsearch.xpack.inference.qa.mixed.MixedClusterSpecIT.bwcVersion;
+import static org.hamcrest.Matchers.empty;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.not;
+
+public class AzureOpenAiServiceMixedIT extends BaseMixedIT {
+
+    private static final String OPEN_AI_AZURE_EMBEDDINGS_ADDED = "8.14.0";
+
+    private static MockWebServer openAiEmbeddingsServer;
+
+    // @BeforeClass test mute
+    public static void startWebServer() throws IOException {
+        openAiEmbeddingsServer = new MockWebServer();
+        openAiEmbeddingsServer.start();
+    }
+
+    // @AfterClass test mute
+    public static void shutdown() {
+        openAiEmbeddingsServer.close();
+    }
+
+    @SuppressWarnings("unchecked")
+    @LuceneTestCase.AwaitsFix(bugUrl = "AzureOpenAI doesn't support webserver URL in tests")
```

Comment:
This should point to an open GitHub issue. If we never intend to run this test then why bother having it?

Reply:
I removed this test for now, we will probably want to add it back once we get it working with the webserver.